README.md

```diff
@@ -38,6 +38,15 @@ Environments under Python 2.7.12 and 3.6.0 are tested.
 
 # Features
 
+## Data conventions
+
+ChainerCV follows the following conventions.
+
++ Channels of a color image are ordered as RGB. The shape of an image-array is CHW (channel, height, width).
++ Axis-dependent values follow row-column order.
+  + Shape of an image is always represented as `(height, width)`.
+  + Coordinates of bounding boxes and keypoints are ordered as `(y, x)`.
```

Comment:
By this description, the coordinates of bounding boxes can be `(y_min, y_max, x_min, x_max)` or `(cy, cx, height, width)`.